<launch>

	<node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf">
	  <param name="output_frame" value="R_005/odom_combined"/>
	  <param name="base_footprint_frame" value="R_005/base_footprint"/>
	  <param name="freq" value="50.0"/>
	  <param name="sensor_timeout" value="1.0"/>  
	  <param name="odom_used" value="true"/>
	  <param name="imu_used" value="false"/>
	  <param name="vo_used" value="true"/>
	  <param name="debug" value="true"/>

	  <remap from="odom" to="R_005/odom" />
	  <remap from="imu_data" to="R_005/t265/imu" />
	  <remap from="vo" to="R_005/odom/sample" />

	</node>

</launch>

